lib/fs.js

```diff
@@ -1585,6 +1585,10 @@ function encodeRealpathResult(result, options, err) {
   }
 }
 
+// This is removed from the fs exports in lib/module.js in order to make
+// sure that this stays internal.
+const realpathCacheKey = fs.realpathCacheKey = Symbol('realpathCacheKey');
```

Comment:
I'm having trouble keeping up with the can and cannot situation of `internal/fs`, but it appears to be there in master right now.